src/components/QueryEditor/MetricAggregationsEditor/SettingsEditor/index.tsx

```diff
@@ -7,7 +7,7 @@ import { extendedStats } from '../../../../queryDef';
 import { MetricAggregation, ExtendedStat } from '../../../../types';
 import { useQuery } from '../../ElasticsearchQueryContext';
 import { SettingsEditorContainer } from '../../SettingsEditorContainer';
-import { isMetricAggregationWithInlineScript, isMetricAggregationWithMissingSupport } from '../aggregations';
```

Comment:
can you clean also the `isMetricAggregationWithInlineScript` in the aggregations file?

Reply:
Done